web/frontend/src/components/servo.vue

```diff
@@ -22,7 +22,7 @@ const stop = () => {
 
 const move = (amount: number) => {
   const servo = props.rawStatus;
-
+  // @ts-expect-error @TODO Proto is incorrectly typing this. It expects servo.positionDeg
   const oldAngle = servo.position_deg ?? 0;
```

Comment:
see [this thread](https://github.com/viamrobotics/rdk/pull/2189#discussion_r1167352440) for context